docs/ERROR_CODES.md

```diff
@@ -102,6 +102,7 @@ This file was **AUTOMATICALLY** generated by
 | 121       | 0x79  | `CHIP_ERROR_INSUFFICIENT_PRIVILEGE`                |
 | 125       | 0x7D  | `CHIP_ERROR_MESSAGE_COUNTER_EXHAUSTED`             |
 | 126       | 0x7E  | `CHIP_ERROR_FABRIC_EXISTS`                         |
+| 127       | 0x7F  | `CHIP_ERROR_ENDPOINT_EXISTS`                       |
```

Comment:
Was this and old deprecated error code we can reuses or should we use a new value?

Reply:
It was a free available error code. unless the code comments were wrong.